src/content/HigherOrderComponents.js

```diff
@@ -22,7 +22,7 @@ export default function Content() {
         <ol>
           <li>
             The <code>ShowTheTime</code> component rendered by <code>App</code>{' '}
-            is an HOC (<code>withCurrentDate</code>)
+            is a HOC (<code>withCurrentDate</code>)
```

Comment:
Not sure which is correct here. In my head, I read this as "is an H O C"

Reply:
I wasn't sure about this too so I checked the React docs, it is "a HOC" there. I guess it depends whether you pronounce the H like "haitch" /heɪtʃ/ or "aitch" /eɪtʃ/.